source/includes/steps-starting-vsce-connect-with-advanced-connection-settings.yaml

```diff
@@ -467,8 +532,34 @@ content: |
   If omitted, defaults to :guilabel:`Primary`. To learn more about read 
   preferences, see :manual:`Read Preference </core/read-preference/>`. 
 
+  You can select additional advanced options :guilabel:`Replica Set Name`, 
+  :guilabel:`Default Authentication Database` and :guilabel:`URI Options` 
+  which specify additional behavior of your MongoDB connection:
```

Comment:
I don't think we need to list out the options in the intro to this table. Maybe we can shorten this whole thing to:

"You can specify additional MongoDB connection behavior with the following options: "